constants.py

```diff
@@ -8,10 +8,15 @@
 # Default port to use when forwarding payload received on the translator's unicast server socket as multicast.
 DEFAULT_MULTICAST_PORT = 9002
 # URL to use when submitting stream information to the Multicast Menu
-MULTICASTMENU_ADD_URL = 'https://multicastmenu.herokuapp.com/add/'
-# Email address to use when submitting stream information to the Multicast Menu. Lenny has OK'ed using his email address
-# until we have a group email.
-MULTICASTMENU_EMAIL = 'lenny@juniper.net'
+# Temporarly until SSL and domain name are set up
+MULTICASTMENU_ADD_URL = 'http://18.209.44.34/add/'
+# URL to use when removing stream information from the Multicast Menu
+# Temporarly until SSL and domain name are set up
+MULTICASTMENU_REMOVE_URL = 'http://18.209.44.34/remove/'
 # Number of worker threads dedicated to submitting stream information to the Multicast Menu.
 MULTICASTMENU_THREADS = 10
+# Unique identifier that is registered with Multicast Menu.
+MULTICASTMENU_UID = "8ZMXL14MLAh0pQ7NTcGYZOCBRrOhg3GaY9X0y4bW"
```

Comment:
Yes and no. It is a UID that identifies the translator's account, but it is not used to control stream access. It is used to generally verify the translator when the translator makes any sort of request. When the translator adds a stream, it will get a unique identifier for managing that stream. For both UIDs, my goal was less authorization and more identification. 

Security wasn't at the top of my thought process when setting this up, but thinking about it now, I agree this definitely shouldn't be hard coded into version control.